Source/SIMPLib/Geometry/SharedEdgeOps.cpp

```diff
@@ -78,7 +78,7 @@ SharedEdgeList::Pointer GEOM_CLASS_NAME::getEdges()
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
-void GEOM_CLASS_NAME::setVertsAtEdge(int64_t edgeId, int64_t verts[2])
+void GEOM_CLASS_NAME::setVertsAtEdge(size_t edgeId, int64_t verts[2])
```

Comment:
I believe verts[2] should be size_t as well, although that will require additional changes.

Reply:
The actual value of the vertex is an int64_t, the index into the array is a size_t.

Reply:
Correct me if I'm wrong, but the value of the vertex is its ID, right?  I'm confused why only some IDs are being updated and some are not.

Reply:
Let me go back and work my way through it again. I'm just not really sure why int64_t was used for ids.